src/lib/flight_tasks/tasks/ManualAltitude/FlightTaskManualAltitude.cpp

```diff
@@ -50,8 +50,32 @@ bool FlightTaskManualAltitude::updateInitialize()
 {
 	bool ret = FlightTask::updateInitialize();
 
-	_sticks.checkAndSetStickInputs(_time_stamp_current);
-	_sticks.setGearAccordingToSwitch(_gear);
+	manual_control_switches_s manual_control_switches;
+
+	if (_manual_control_switches_sub.copy(&manual_control_switches) && (manual_control_switches.timestamp > 0)) {
```

Comment:
TODO: test landing gear (stabilized mode and altitude).

Reply:
Yes I was suggesting testing landing gear with both stabilized and altitude because it's handled in different places.